search/domain/classic_api/query_parser.py

```diff
@@ -0,0 +1,167 @@
+"""
+Utility module for Classic API Query parsing.
+
+Uses lark-parser (EBNF parser) [1].
```

Comment:
The classic API reimplementation has been updated to use lark-parser.

Reply:
:+1: